src/utils/set-base-route-and-business-id.ts

```diff
@@ -22,8 +22,12 @@ export function setBaseRouteAndBusinessId (pathname: string, processEnvBaseUrl:
     sessionStorage.setItem('TEMP_REG_NUMBER', id)
     // ensure we don't already have a Business ID in scope
     sessionStorage.removeItem('BUSINESS_ID')
+    // If user hits just root with no other info, they are sent back to BRD
   } else {
-    throw new Error('Missing or invalid Business ID or Temporary Registration Number.')
+    // Make sure we don't have anything already in scope
+    sessionStorage.removeItem('BUSINESS_ID')
+    sessionStorage.removeItem('TEMP_REG_NUMBER')
+    return
```

Comment:
OK. Thanks for the comment.
Please add another comment indicating how/where this condition is handled.

Reply:
I was looking for a code comment 😄 

Now that I think about it, bootstrap businesses (Txxx) never do anything in Filings UI, right? So that's a bit more code that can be cleaned up.